site2/docs/getting-started-standalone.md

```diff
@@ -104,34 +104,34 @@ bin/pulsar-client consume my-topic -s 'my-subscription' -p Earliest -n 0
 
 Earliest means consuming from the earliest **unconsumed** message. `-n` configures the number of messages to consume, 0 means to consume forever.
 
-As before, this is useful for trialling things on the command line, but in practice you'll use the Consumer API in your application code, or Pulsar IO connectors for reading data from Pulsar to push to other systems.
+As before, this is useful for experimenting with messages, but in practice you'll use the Consumer API in your application code, or Pulsar IO connectors for reading data from Pulsar to push to other systems.
 
-You'll see the messages that you produce in the previous step:
+You'll see the messages you produce in the previous step:
 
 ```text
 ----- got message -----
 key:[null], properties:[], content:Hello Pulsar!
 ```
 
-## Step 6. Write some more messages
+## Write some more messages
 
 Leave the consume command from the previous step running. If you've already closed it, just re-run it.
 
-Now open a new terminal window and produce more messages, the default message separator is `,`:
+Now open a new terminal window and produce more messages. The default message separator is `,`:
 
 ```bash
 bin/pulsar-client produce my-topic --messages "$(seq -s, -f 'Message NO.%g' -t '\n' 1 10)"
 ```
 
 Note how they are displayed almost instantaneously in the consumer terminal.
 
-## Step 7. Stop the Pulsar cluster
+## Stop the Pulsar cluster
 
 Once you've finished you can shut down the Pulsar cluster. Press **Ctrl-C** in the terminal window in which you started the cluster.
 
-## Further readings
+## Related Topics
 
-* Read [Pulsar Concepts and Architecture](concepts-architecture-overview.md) to learn more about Pulsar fundamentals.
-* Read [Pulsar Client Libraries](client-libraries.md) to connect Pulsar with your application.
-* Read [Pulsar Connectors](io-overview.md) to connect Pulsar with your existing data pipelines.
-* Read [Pulsar Functions](functions-overview.md) to run serverless computations against Pulsar.
+[Pulsar Concepts and Architecture](concepts-architecture-overview.md)
+[Pulsar Client Libraries](client-libraries.md)
+[Pulsar Connectors](io-overview.md)
+[Pulsar Functions](functions-overview.md)
```

Comment:
I wonder why to make these changes.

Reply:
Also, this section outside of a list can be rendered mangled together.

Reply:
Hello tisonkun! This change makes the formatting consistent with the pages being developed under the new Information Architecture.  The last section is Related Topics. The topic titles are then listed as hyperlinks.

Reply:
Thanks for your explanation. I don't have a strong feeling about these wording changes. Although, here is a screenshot to explain my comment "mangled content":

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/18818196/189150155-2ecfe9b1-dd6d-45b2-90d4-fb90cd550693.png">

Still, I suggest to separate changes on getting-started-standalone page in a dedicated PR so that we can handle these two topics independently.

Reply:
So I just suggest you use the unordered list to format the Related topics candidates.

Reply:
Thanks, tisonkun - I now understand your concerns about the unordered list.  I'll make those changes today.  Thank you!